Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -19,6 +19,8 @@ const LoginView = () =>
     import(/* webpackChunkName: "login" */ "@/views/login.vue");
 const LogoutView = () =>
     import(/* webpackChunkName: "logout" */ "@/views/logout.vue");
+const LogoutCompleteView = () =>
+    import(/* webpackChunkName: "logout" */ "@/views/logoutComplete.vue");
```

Comment:
The webpackChunkName doesn't match the name of the view.

Reply:
Thats ok, that would put both on the same chunk